examples/shaders/shaders_basic_pbr.c

```diff
@@ -94,9 +94,10 @@ int main()
     camera.fovy = 45.0f;                                // Camera field-of-view Y
     camera.projection = CAMERA_PERSPECTIVE;             // Camera projection type
 
-    // Load PBR shader and setup all required locations
+    // Load the PBR shader and setup all required locations
```

Comment:
I'm afraid I'm not using `the` in any of the shaders examples, for consitency, I prefer to keep it without it.